pkg/server/commands/reverseexpand/reverse_expand_weighted.go

```diff
@@ -682,11 +682,6 @@ func (c *ReverseExpandQuery) intersectionHandler(
 		return fmt.Errorf("%w: operation: intersection: %s", ErrLowestWeightFail, err.Error())
 	}
 
-	if !intersectionEdgeComparison.DirectEdgesAreLeastWeight && intersectionEdgeComparison.LowestEdge == nil {
```

Comment:
Maybe a quick note here in the review why this is removed?

Reply:
The weighted graph no longer supports models that can hit this path